x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/DriverEarlyTerminationException.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.compute.operator;
+
+final class DriverEarlyTerminationException extends RuntimeException {
```

Comment:
I have removed these changes. Can you revert them?

Reply:
I am not sure I understand this - if we remove those changes, how the interrupt mechanism works? Let's discuss it.